src/test/java/org/jabref/logic/layout/format/AuthorsTest.java

```diff
@@ -61,6 +61,50 @@ public void testStandardUsageSix() {
                 a.format("Bob Croydon Bruce and Charles Manson and Jolly Jumper and Chuck Chuckles"));
     }
 
+    /**
+     * Test the FirstFirst method in authors order.
+     * setArgument() will pass the String into handleArgument() to set flMode.
+     * Increase branch coverage from 58% to 61%.
+     */
+    @Test
+    public void testStandardUsageFirstFirst() {
+        ParamLayoutFormatter a = new Authors();
+        a.setArgument("FirstFirst, Comma, Comma");
+        assertEquals("B. C. Bruce, C. Manson, J. Jumper",
+                a.format("Bob Croydon Bruce and Charles Manson and Jolly Jumper"));
+    }
+
+    /**
+     * Test the And method in separators of authors.
+     * setArgument() will pass the String into handleArgument() to set separator and lastSeparator.
+     * Increase branch coverage from 58% to 66%.
+     */
+    @Test
+    public void testStandardUsageAnd() {
+        ParamLayoutFormatter a = new Authors();
+        a.setArgument("fullname, LastFirst, Comma, And");
+        assertEquals("Bruce, Bob Croydon, Jumper, Jolly and Manson, Charles", 
+                a.format("Bob Croydon Bruce and Jolly Jumper and Charles Manson"));
+        
+        a = new Authors();
+        a.setArgument("fullname, LastFirst, Add, And");
```

Comment:
Should this be `And` for both, or should it be `Add` for one of them? Just want to check :)

Reply:
Yes! It should be `And`. I will change it in the next commit.